docroot/themes/humsci/su_humsci_theme/js/su_humsci_theme.js

```diff
@@ -2,6 +2,11 @@
   'use strict';
   Drupal.behaviors.suHumSciTheme = {
     attach: function (context, settings) {
+      var headings = ['h2', 'h3', 'h4', 'h5', 'h6'];
```

Comment:
Why are you attaching these classes with javascript?

Reply:
can't apply css based on children of an element.

Reply:
Where are you unable to control the markup that you cannot wrap a heading in a link?

Reply:
depending on what the user configures in the ui for field markup.

Reply:
Sounds like we need field level components (atoms)